react/features/prejoin/middleware.js

```diff
@@ -68,12 +69,20 @@ MiddlewareRegistry.register(store => next => async action => {
     }
 
     case SET_AUDIO_MUTED: {
-        store.dispatch(setPrejoinAudioMuted(Boolean(action.muted)));
+        // Don't forward mute audio action if not on prejoin page
+        if (isPrejoinPageVisible(store.getState())) {
```

Comment:
Maybe this check should be done first thing, before the `switch`.

Reply:
I can't do this because there is an async action depending on the `isPrejoinPageVisible` selector.